<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configuration_widget</class>
 <widget class="QWidget" name="configuration_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QStackedWidget" name="stackedWidget_config">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_settings">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_settings_profiles">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_settings_identifier"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_settings_energizations">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_settings_lTypes">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_settings_cbTypes">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_settings_units">
         <item>
          <property name="text">
           <string>Metric</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Imperial</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_settings_frequency">
         <item>
          <property name="text">
           <string>AC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DC</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_settings_coatings">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_settings_cTypes">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Identifier</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Units</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Frequency</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string># of Lead Types</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string># of Coatings</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string># of Energizations</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string># of Conductor Types</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string># of Cable Types</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string># of Profiles</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_lTypes">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_lTypes_name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="comboBox_lTypes_chooser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_page">
     <item>
      <property name="text">
       <string>Configuration Settings</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lead Types</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Coatings</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Energizations</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tolerances</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Conductor Types</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cable Types</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Computations</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Profiles</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
